packages/loader/container-loader/package.json

```diff
@@ -11,122 +11,64 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/index.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
 				"types": "./dist/index.d.ts",
 				"default": "./dist/index.js"
 			}
 		},
-		"./test/container": {
-			"import": {
-				"types": "./lib/container.d.mts",
-				"default": "./lib/container.mjs"
-			},
-			"require": {
-				"types": "./dist/container.d.ts",
-				"default": "./dist/container.js"
-			}
-		},
-		"./test/contracts": {
-			"import": {
-				"types": "./lib/contracts.d.mts",
-				"default": "./lib/contracts.mjs"
-			},
-			"require": {
-				"types": "./dist/contracts.d.ts",
-				"default": "./dist/contracts.js"
-			}
-		},
```

Comment:
These removals do not look right to me. What is happening with this package?

Reply:
Added `/test` exports back